website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -14,18 +14,22 @@ dbt "adapters" are responsible for _adapting_ dbt's functionality to a given dat
 This guide will walk you through the first two steps, and provide some resources to help you validate that your new adapter is working correctly.
 
 ## Scaffolding a new adapter
+To create a new adapter plugin you can use one of two strategies usually.
+1. If your adapter is forked from or compaitiable with a pre exisiting adapter ex. (postgres => redshift, dbt-spark => dbt-databricks) you may be able to tap into a lot of preexisting functionality already cutting a lot of setup from scratch work by creating a wrapper around the existing adapter plugin.
```

Comment:
@jtcohen6 do we recommend this option to our users? If so, we don't give users a descriptive way to do this. Would we want to add that?

Reply:
I think it is needed, but would require more engineering to make happen (if we want the experience to be as solidy as as the 2nd strategy is). I still think a template for simple inheiritance here's [dbt-synapse's __init__.py](https://github.com/dbt-msft/dbt-synapse/blob/master/dbt/adapters/synapse/__init__.py). The classy thing would be a way to pull the parent class's methods inot the scaffold, but as a first rev, we could just have the inheiritance with no commented-out classes.